lib/stm32/common/desig_common_all.c

```diff
@@ -66,15 +53,18 @@ void desig_get_unique_id_as_string(char *string,
  * @param string pointer to store serial in, must be at least 13 bytes
  */
 void desig_get_unique_id_as_dfu(char *string) {
-	uint8_t *id = (uint8_t *)DESIG_UNIQUE_ID_BASE;
+	uint32_t uid_buf[3];
+	uint8_t *uid = (uint8_t *)uid_buf;
+
+	desig_get_unique_id(uid_buf);
 
 	uint8_t serial[6];
-	serial[0] = id[11];
-	serial[1] = id[10] + id[2];
-	serial[2] = id[9];
-	serial[3] = id[8] + id[0];
-	serial[4] = id[7];
-	serial[5] = id[6];
+	serial[0] = uid[11];
+	serial[1] = uid[10] + uid[2];
+	serial[2] = uid[9];
+	serial[3] = uid[8] + uid[0];
+	serial[4] = uid[7];
+	serial[5] = uid[6];
```

Comment:
same here, it might be "nice" but it just makes churn for no real particular reason in unrelated code.

Reply:
This is just to make them match the above.